client/lib/plans/plans-list.js

```diff
@@ -384,7 +385,7 @@ const getPlanJetpackSecurityDailyDetails = () => ( {
 				'Great for brochure sites, restaurants, blogs, and resume sites.'
 		),
 	getTagline: () =>
-		isEnabled( 'plans/alternate-selector' )
+		getJetpackCROActiveVersion() === 'v1'
```

Comment:
Should we be including the `v2` variant in this check as well? (Similar to the pricing header)

Reply:
I almost did that but then I noticed that v2 uses v0's copy in this case.